template/layout.eml

```diff
@@ -1,10 +1,16 @@
-type tab = 
+type tab =
   | Home
   | Blog
   | Docs
   | Api
   | Community
 
+let extra_link = function
+  | Blog ->
```

Comment:
Maybe it'd be useful to put it on all pages so that even people visiting the home page are able to discover the RSS feed ?

I see that https://dri.es is doing it so it looks like standard practice.

Reply:
Yes, I agree. You can put multiple rss feed links in the header. Just use an informative title :-)